bee-common/src/lib.rs

```diff
@@ -0,0 +1,13 @@
+// Copyright 2020 IOTA Stiftung
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+// the License. You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+// an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and limitations under the License.
+
+/// A logger backend for the `log` crate.
```

Comment:
Is it ? The rust std library uses `///`.

Reply:
Imagine that the documentation gets several lines long as the complexity or need for documentation might increase. And also imagine that you have several modules. Then by your way it would end up in a lot of documentation in the file that imports them. IMHO module documentation should happen within the module so it automatically moves wherever the module might move in the future. This prevents us from having to edit multiple files in such case.

Reply:
Alright 👍